icu4c/source/i18n/formattedval_sbimpl.cpp

```diff
@@ -154,7 +154,10 @@ bool FormattedValueStringBuilderImpl::nextPositionImpl(ConstrainedFieldPosition&
         if (i > fString.fZero && prevIsSpan) {
             int64_t si = cfpos.getInt64IterationContext() - 1;
             U_ASSERT(si >= 0);
-            if (_field == Field(UFIELD_CATEGORY_LIST, ULISTFMT_ELEMENT_FIELD)) {
+            int32_t previ = i - spanIndices[si].length;
```

Comment:
should this variable be prevI instead of previ if we follow the naming case rule?

Reply:
Yes, except that I don't apply that style specifically to single-character variable names, and especially `i`, since capital `I` is confusable with lowercase `l`.